README.md

```diff
@@ -30,3 +30,88 @@ The aims of the project are:
 3.	Calculate a ratio between public and private transport journey times 
 4.	Calculate an overall metric for the entire city
 5.	Compare the results for each city
+
+### Using VS code
+
+#### Create new branch 
+
+Initially created a branch on GitHub from the develop branch. 
+Within the terminal:
+```python 
+git fetch -p
+``` 
+This will update your local branches in line with remote branches
+Within VS code you can switch branches using the button in the bottom left corner. 
```

Comment:
Link to the official instructions on how to switch branches. https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_branches